src/sections/CompanyTeamSection/index.css

```diff
@@ -14,3 +14,44 @@
     margin-top: $spacing-28;
   }
 }
+
+.team {
+  display: grid;
+
+  grid-column-gap: $spacing-28;
+  grid-row-gap: $spacing-112;
+  grid-template-columns: 1fr 1fr 1fr;
+
+  margin-top: $spacing-112;
+
+  @include Breakpoint-tabletAndBelow {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @include Breakpoint-mobileOnly {
+    grid-auto-rows: 1fr;
+    grid-row-gap: $spacing-28;
+    grid-template-columns: 1fr;
+  }
+}
+
+.challenge {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  padding: $spacing-56;
```

Comment:
Can you confirm this is correct. Looking at figma, the padding it's 40px and 28px mobile

Reply:
Yeah padding is smaller on mobile. But the 40 spacings are actually 56, Ines said it

Reply:
👍